packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -176,6 +177,16 @@ export class VaTextInput {
    */
   @Prop() currency?: boolean = false;
 
+   /**
+   * This property displays a prefix that accepts either a string or an object with an icon string.
+   */
+   @Prop() inputPrefix?: string;
```

Comment:
> an object with an icon string
What does this mean? Is there a certain format that needs to be followed in order to use a custom icon?

Reply:
This won't be an issue any more as I created a new prop.